packages/google_maps_flutter/google_maps_flutter_android/README.md

```diff
@@ -78,6 +78,10 @@ Google Play the latest renderer will not be available and the legacy renderer wi
 WARNING: `AndroidMapRenderer.legacy` is known to crash apps and is no longer supported by the Google Maps team
 and therefore cannot be supported by the Flutter team.
 
+#### Cloud-based map styling
+
+Cloud-based map styling works on Android only if `AndroidMapRenderer.latest` map renderer has been initialized.
```

Comment:
I think people may read this as having to explicitly initialize the `latest` renderer, but in most cases it's the default now. How about "Cloud-based map styling is not supported with the `legacy` renderer."

Reply:
Done.